src/modules/tasks/task-list/tasks.html

```diff
@@ -98,22 +98,22 @@ <h1>No Tasks Detected</h1>
     </center>
     <div ng-if="taskCntrl.isDataLoading" class="spinner spinner-lg"></div>
     <div class="flex-table list-group list-view-pf list-view-pf-view task-list-table" ng-show="!taskCntrl.isDataLoading">
-        <div class="ft-row list-group-item" ng-repeat="task in taskCntrl.filteredTaskList = (taskCntrl.taskList | filter: taskCntrl.searchBy | filter:taskCntrl.filterByStatus | filter: taskCntrl.filterByCreatedDate| orderBy: ['-created_at', 'job_id'])">
+        <div class="ft-row list-group-item" ng-repeat="task in taskCntrl.filteredTaskList = (taskCntrl.taskList | filter: taskCntrl.searchBy | filter:taskCntrl.filterByStatus | filter: taskCntrl.filterByCreatedDate| orderBy: ['-createdAt', 'jobId'])">
             <div class="ft-column ft-icon ">
-                <i ng-class="{ 'pficon pficon-error-circle-o': (task.status==='failed' ), 'pficon pficon-ok': (task.status==='finished' ), 'pficon pficon-warning-triangle-o': (task.status==='warning' ), 'fa fa-spinner': (task.status==='processing' || task.status==='new' )} "></i>
+                <i ng-class="{ 'pficon pficon-error-circle-o': (task.status==='Failed' ), 'pficon pficon-ok': (task.status==='Completed' ), 'pficon pficon-warning-triangle-o': (task.status==='Completed with Errors' ), 'fa fa-spinner': (task.status==='Processing' || task.status==='new' )} "></i>
```

Comment:
{ 'pficon pficon-error-circle-o': (task.status==='Failed' ), 'pficon pficon-ok': (task.status==='Completed' ), 'pficon pficon-warning-triangle-o': (task.status==='Completed with Errors' ), 'fa fa-spinner': (task.status==='Processing' || task.status==='new' )}

write it in the controller and return the value. It will be a much cleaner way.

Reply:
Acknowledge